src/event-sources/aws/sqs.js

```diff
@@ -1,18 +1,24 @@
 const getRequestValuesFromSqs = ({ event }) => {
-  const method = 'POST'
-  const headers = { host: 'sqs.amazonaws.com' }
-  const body = event
+  const method = "POST";
+  const headers = { host: "sqs.amazonaws.com" };
+  const body = event;
 
   return {
     method,
     headers,
-    body
-  }
-}
+    body,
+  };
+};
 
-const getResponseToSqs = ({ body }) => JSON.parse(body)
+const getResponseToSqs = ({ statusCode, body }) => {
+  if (statusCode >= 400) {
+    throw new Error(body);
```

Comment:
Could you explain this change please? I want to ensure you can still respond with `batchItemFailures`

Reply:
When using HTTP server for handling the event source, the server may response the 5xx or 4xx http-code for any exception. This changing will useful for the case we using the same source code for handling http and event-sources.

Reply:
Let's revert this. I don't think we should throw errors at this level based on http code.

Reply:
I reverted